alibaba-dragonwell/content.md

```diff
@@ -0,0 +1,42 @@
+## Overview
+
+The images in this repository contain Alibaba Dragonwell binaries built by [JDK Team at Alibaba Cloud](https://github.com/alibaba/dragonwell8).
+
+# What is Alibaba Dragonwell ?
+
+Alibaba Dragonwell, as a downstream version of OpenJDK, is the OpenJDK implementation at Alibaba optimized for online e-commerce, financial, and logistics applications running on 100,000+ servers. Alibaba Dragonwell is the engine that runs these distributed Java applications in extreme scaling.
+
+# Do you provide JRE (Java Runtime Environment) Images?
+
+Current there are only JDK (Java Developer Kit) images.
+
+# How to use this Image
+
+To run a pre-built jar file with the latest OpenJDK 11, use the following Dockerfile:
+
+```dockerfile
+FROM %%IMAGE%%:11
+RUN mkdir /opt/app
+COPY japp.jar /opt/app
+CMD ["java", "-jar", "/opt/app/japp.jar"]
```

Comment:
Alternatively, you could add `WORKDIR` and make this even shorter:

```suggestion
WORKDIR /opt/app
COPY japp.jar .
CMD ["java", "-jar", "japp.jar"]
```

(Ultimately up to you / your preference)